mlflow/tracking/_tracking_service/client.py

```diff
@@ -617,6 +625,14 @@ def log_artifact(self, run_id, local_path, artifact_path=None):
         else:
             artifact_repo.log_artifact(local_path, artifact_path)
 
+    def download_trace(self, request_id: str) -> Dict[str, Any]:
```

Comment:
Should this be kept?

Reply:
The request ID shouldn't be in the API signature for `upload_trace()` or `download_trace()`, right? We should be able to identify the request ID by parsing the artifact URI

Reply:
Sorry, ignore my last comment if you saw it. No, we shouldn't keep this.